tensorflow/compiler/mlir/xla/tests/legalize-tf.mlir

```diff
@@ -1711,6 +1711,30 @@ func @leaky_relu_grad(%arg0: tensor<1x4x4xf32>, %arg1: tensor<1x4x4xf32>) -> ten
     return %0 : tensor<1x4x4xf32>
 }
 
+// CHECK-LABEL: func @softsign
+func @softsign(%arg0: tensor<4x10xf32>) -> tensor<4x10xf32> attributes {tf.entry_function = {}} {
```

Comment:
The attribute does not seem necessary here

Reply:
Thanks for pointing this out.